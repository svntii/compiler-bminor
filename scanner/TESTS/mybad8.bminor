square: function integer ( x: integer ) = {
	return "1111111111111111122222222222222222222222222222222222222222222222222222222222222222222200000000000000000000033030303301000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000";
}