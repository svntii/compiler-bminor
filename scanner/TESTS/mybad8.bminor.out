IDENTIFIER		text: square
COLON		text: :
KEYWORD		text: function
KEYWORD		text: integer
PARL		text: (
IDENTIFIER		text: x
COLON		text: :
KEYWORD		text: integer
PARR		text: )
ASSIGNMENT		text: =
CRLL		text: {
KEYWORD		text: return
